### PR TITLE
Add General Centroid Code

### PR DESCRIPTION
**Pull request by nirvanarsc**
Hello, I've fixed the Java general Centroid code for https://usaco.guide/plat/centroid?lang=java. But I don't know c++, so maybe someone else can reference this and update the c++ code 😛 

I have a successful submission with that code here https://codeforces.com/contest/342/submission/113088188

### CODE REVIEW

**Comment by nirvanarsc** [comment]
> Hey there! 👋 This repository uses the [Prettier](https://prettier.io) code style.
> 
> You forgot to format these files in your pull request:
> 
> * content/5_Plat/Centroid.mdx
> 
> The [Prettier installation guide](https://prettier.io/docs/en/install.html) is a good place to get started with formatting your code properly.
> Thanks!
> 
> ❤️
> 
> Your friendly [Prettifier bot](https://www.prettifier.io).

Screen Shot 2021-04-15 at 21 56 58
Maybe I'm missing something, is this not how you run prettier?

**Comment by bqi343** [comment]
> Maybe I'm missing something, is this not how you run prettier?

@thecodingwizard ?

**Comment by thecodingwizard** [comment]
Either works. You can also do yarn prettier --write filepath

File: content/5_Plat/Centroid.mdx [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: 'Centroid Decomposition'
 author: Siyong Huang, Benjamin Qi
 prerequisites:
   - intro-tree
-description: 'Decomposing a tree to faciliate path computations.'
+description: 'Decomposing a tree to facilitate path computations.'
```
Comment (frodakcin) [inline content/5_Plat/Centroid.mdx:7]: Nice catch!

**Review by frodakcin — CHANGES_REQUESTED** [review]
Looks good. Just one thing: can you switch to tabs instead of spaces in the sample code?

**Comment by nirvanarsc** [comment]
>Either works. You can also do yarn prettier --write filepath

I see thanks just asking cause I got the message from the bot that I didn't format the file, but then running the command says its already formatted well

>Looks good. Just one thing: can you switch to tabs instead of spaces in the sample code?

Updated could you take another look 🙇

**Review by frodakcin — APPROVED** [review]

**Pull request merged**